css/css-backgrounds/border-image-width-010.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Backgrounds and Borders Test: border image area - border-image-width with 2 values with percentage unit</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-width">
+  <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+
+  <meta name="flags" content="">
+  <meta content="This test checks that the border image area can still exists even if the border area (or border belt) of an element is inexistent. The difference with this test and the border-image-width-005 test is the usage of percentage unit. In this test, the border image fills the content area, the padding area and 25px of the extended-beyond-border-belt area, 25px into the margin area of the element." name="assert">
+
+  <!--
+  created 2015-11-18
+  [css-backgrounds] Please remove the "unless there is a border image" wording
+  http://lists.w3.org/Archives/Public/www-style/2015Nov/0258.html
+  -->
+
+  <style>
+  p
+    {
+      margin-bottom: 41px; /* border-image-outset + 16px (1em) .
+      This 41px margin-bottom value is just to position the rendered layout to accurately match the reference file . */
+    }
+
+  div
+    {
+      background-color: red;
+      border-color: red;
+      border-style: none; /* or border-style: dotted or any other 'border-style' value */
+      border-width: 0px; /* so the border belt (or border area) of the element is 0px tall and 0px wide */
+      border-image-source: url("support/outline-5px-10px-15px-20px-green.png");
+      border-image-slice: 5% 10% 15% 20%;
+      /* <number> Percentages are relative to the size of the image: the width of the image for the horizontal offsets, the height for vertical offsets. */
+      border-image-width: 70% 30%;
+      /* The four values of 'border-image-width' specify offsets that are used to divide the border image area into nine parts. They represent inward distances from the the top, right, bottom, and left sides of the area, respectively. In this test, the border-image-width divide the border image area into four (4) parts, not nine parts. The center (fill) area is 0px wide by 0px tall. */
```

Comment:
This comment is incorrect. This case has 6 parts, not 4. You would be able to see this more clearly if you did not use a solid color image, but rather one with a pattern.

Reply:
Correct. I have made such correction in the new meta-test.